### ISSUE TITLE
Some extra functions

### ISSUE THREAD

**Issue by c-sanchez** [Nain57/Smart-AutoClicker#354]
1. **Clear the clipboard.**
I currently use ClearClipboard along with Smart AutoClicker.
https://github.com/amnesica/ClearClipboard
Since ClearClipboard has no interface and simply clears the clipboard on startup, I can use it without problems with an Intent.
However I can see that there is a small delay before continuing the next action, I guess because the app must be started first in any case.
If Smart AutoClicker had this as another function it would be great, since there would be no need to have an additional application for this task.

2. **Switch between apps.**
It would be very useful to automate tasks that involve more than one application, such as copying data from one site to another.

3. **Switch between the last two recent apps**
On Android it is possible to switch between the last two applications almost immediately by simply pressing the recent apps button twice quickly.
Making it unnecessary to wait for the menu with the list of recent apps to be displayed.
_(I am surprised that few people know about this behavior when I think it is really cool and useful)_
![Android 3 Buttons Navbar](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ8UnCMC_vY_9jP33tvyIbklM0_axAMbUzUCB2KDCObTC9hrmY5)
It would also be useful to be able to use this function directly from Smart AutoClicker if possible, especially when using the system with Gestures instead of the 3 buttons navbar.

4. **Enter text**
Useful if you need to enter some predetermined text at a specific time.
Currently it is only possible by configuring Smart AutoClicker in some way to interact with the keyboard, Which is really tedious to set up perfectly.
Also could malfunction or stop working with any slight change, such as keyboard size, color, etc.
![GBoard-SwiftKey](https://tech-latest.com/wp-content/uploads/2020/05/Gboard-Vs-SwiftKey-Word-Prediction-1024x441.jpg)

5. **Play Sound / Beep**
Could be useful if some type of audible notification is desired in addition to the visual.

**Comment by c-sanchez** [comment]
Edit: Actually calling the apps by intents does restore the app as expected if it is in the recent apps list.

The problem I had was with Binance, which has this activity by default:
" **com.binance.dev/com.eaas.launcher.activities.main.MainActivity** "

Using [Activity Launcher](https://f-droid.org/en/packages/de.szalkowski.activitylauncher/) to view Binance activities and testing them, I found that the correct one to restore the application correctly is this one:
" **com.binance.dev/com.binance.dev.open.BinanceEntryActivity** "

So in summary, Smart AutoClicker can open/switch between applications without any problems just by using simple Intents :)
